homeassistant/components/qnap/manifest.json

```diff
@@ -1,9 +1,12 @@
 {
   "domain": "qnap",
   "name": "QNAP",
+  "config_flow": true,
+  "dependencies": [],
   "codeowners": [],
   "documentation": "https://www.home-assistant.io/integrations/qnap",
   "iot_class": "local_polling",
   "loggers": ["qnapstats"],
+  "integration_type": "device",
```

Comment:
You should create a separate standalone PR, just to set the `integration_type`.

Reply:
Opened PR #93915